spec/forms/gpo_verify_form_spec.rb

```diff
@@ -119,7 +119,7 @@
     end
   end
 
-  describe '#submit' do
+  describe '#submit(is_enhanced_ipp)' do
```

Comment:
Nit: I don't see other [form](https://github.com/18F/identity-idp/blob/main/spec/forms/idv/address_form_spec.rb#L33) [specs](https://github.com/18F/identity-idp/blob/main/spec/forms/idv/phone_form_spec.rb#L22) including the params in the describe block headings.  Is it expected for us to include this?

Reply:
No. I think that is a result of fine/replace.  I deleted it. Resolved with commit 69016c5c359c36009a2d706db169280d86b15b4f